### DISCUSSION TITLE
How to display Latex cell in markdown table?

### DISCUSSION THREAD

**Discussion by wqw547243068** [community/community#32281] [Other Feature Feedback, Questions, & Ideas]
## Questions
- I want to add a cell with Latex code to table in markdown format
- However, Jekyll can't display Latex formula, any solutions for this 

```markdown
|latex|formula|
|---|---|
|\frac|$\frac{1}{2}$|
|\mathbb{SAMPLE}|$\mathbb{SAMPLE}$|
```

GitHub markdown looks normal

|latex|formula|
|---|---|
|\frac|$\frac{1}{2}$|
|\mathbb{SAMPLE}|$\mathbb{SAMPLE}$|

While [Jekyll Page](https://wqw547243068.github.io/2015/02/16/latex-editor/#13%E5%A6%82%E4%BD%95%E8%BF%9B%E8%A1%8C%E5%AD%97%E4%BD%93%E8%BD%AC%E6%8D%A2) couldn't display 
- <img width="184" alt="image" src="https://user-images.githubusercontent.com/3257257/189080924-cc20f921-0437-422a-bcd2-9781cd590193.png">
- Besides, I use MathJax to render Latex

**Comment by yoannchaudet** [comment]
There is no straightforward solution as Jekyll does not support that out of the box but you can get it to work:

1. Change your Pages source to GitHub Actions (this is in beta)
2. Add a "vanilla" Jekyll workflow to your repository ([example](https://github.com/actions/starter-workflows/blob/main/pages/jekyll.yml))
3. Use this plugin: https://github.com/jeffreytse/jekyll-spaceship
